src/e-backport.js

```diff
@@ -91,7 +91,13 @@ program
       `Failed to checkout new branch "${manualBpBranch}"`,
     );
 
-    spawnSync(config, 'yarn', ['install'], gitOpts, `Failed to do "yarn install" on new branch`);
+    spawnSync(
+      config,
+      'yarn',
+      ['install', '--immutable'],
```

Comment:
I think this tries to use `--immutable` on the electron repo which won't have yarn@v4 yet?

Reply:
Ah right 🤦 I will remove these